src/utils/get_last_visit_time.js

```diff
@@ -0,0 +1,32 @@
+export default function getLastVisitTime(timestamp) {
+  if (!timestamp) {
+    return null;
+  }
+  timestamp *= 1000;
+  const now = Math.round(Date.now() / 1000) * 1000;
+  const todayStart = new Date().setHours(0, 0, 0, 0);
+  const yesterdayStart = todayStart - 24 * 60 * 60 * 1000;
+  const visitDate = new Date(timestamp);
+
+  let baseMessage = "last visited ";
+  const options = {
+    hour: "2-digit",
+    minute: "2-digit",
+  };
+  if (timestamp >= todayStart && timestamp <= now) {
+    baseMessage += "at " + visitDate.toLocaleTimeString("uk-UA", options);
```

Comment:
can we get rid of specific local and use default browser local, so users from any country can use it